transports/transport.c

```diff
@@ -1,4 +1,4 @@
-/*! \file   transport.h
+/*! \file   transport.c
```

Comment:
This belongs in a separate commit

Reply:
Separate commit/PR?

Reply:
Yes, it's should be in a separate commit as it's unrelated to `g_malloc0` changes.